modules/ROOT/pages/osv2-faq.adoc

```diff
@@ -25,6 +25,14 @@ Mule 3.8.5 and above, and all 4.x versions.
 
 ====
 
+== What happens when the transaction limit is reached?
+
+Object Store v2 allows up to 100 million transactions for Object Store v2 for an Anypoint Platform organization. This limit is applied to both Production and non-Production environments.
```

Comment:
One Object Store v2 subscription pack allows up to 100 million transactions per month for an Anypoint Platform organization.  This limit is applied across all Production and non-Production environments.

Reply:
Fixed as requested except that I changed "One" to "An"